modules/renter/hostdb/hostdb.go

```diff
@@ -16,10 +21,32 @@ const (
 	scanPoolSize = 1000
 )
 
+var (
+	errNilCS     = errors.New("cannot create renter with nil consensus set")
+	errNilWallet = errors.New("cannot create renter with nil wallet")
+	errNilTpool  = errors.New("cannot create renter with nil transaction pool")
+)
+
 // The HostDB is a database of potential hosts. It assigns a weight to each
 // host based on their hosting parameters, and then can select hosts at random
 // for uploading files.
 type HostDB struct {
+	// modules
+	wallet modules.Wallet
+	tpool  modules.TransactionPool
+
+	// resources
+	log *log.Logger
+
+	// variables
+	blockHeight   types.BlockHeight
+	contracts     map[types.FileContractID]hostContract
+	cachedAddress types.UnlockHash // to prevent excessive address creation
+
+	// constants
+	entropy    [32]byte
+	persistDir string
```

Comment:
I don't think it makes sense to separate variables and constants. Or rather, I don't think it fits the general organizational style of the rest of our large objects. Most are sorted by how they integrate with the package, and not by their specific behavior. (IE separating the modules from the resources)